src/database/models/room.js

```diff
@@ -30,6 +32,11 @@ export default (sequelize, DataTypes) => {
       bedType: DataTypes.STRING,
       bedCost: DataTypes.STRING,
```

Comment:
rename to room cost or something that makes sense cause we book a whole room not only a bed